src/Command/AbstractCommand.php

```diff
@@ -34,20 +32,24 @@ abstract class AbstractCommand extends Command
     protected $annotationReader;
 
     /**
-     * AbstractCommand constructor.
-     *
-     * @param EntityManager          $entityManager
-     * @param Reader|AttributeReader $annotationReader
+     * @var Encrypt
+     */
+    protected $service;
+
+    /**
+     * @return void
      */
     public function __construct(
         EntityManagerInterface $entityManager,
         $annotationReader,
-        DoctrineEncryptSubscriber $subscriber
+        DoctrineEncryptSubscriber $subscriber,
+        Encrypt $service
     ) {
```

Comment:
Adding a non-optional argument to a constructor of an extendable class is a backwards-compatibity-problem.

More information can be found in the [symfony-docs](https://symfony.com/doc/current/contributing/code/bc.html#changing-classes) on this.

Same goes for the renaming of the method lower below.

I understand the need for this as we will want to be using that service indeed in any other encrypt-commands.

I think this also calls for a major version-bump, and that we should document it in `UPGRADING.md` so people upgrading to the new version are aware.